tasks/testUpgrade.ts

```diff
@@ -0,0 +1,69 @@
+import { HardhatRuntimeEnvironment } from 'hardhat/types/runtime'
+
+require('@openzeppelin/hardhat-upgrades')
+require('dotenv').config()
+
+import raft from '../test/abis/latest/Raft.json'
+const raftAbi = raft.abi
+const raftBytecode = raft.bytecode
+
+import specDataHolder from '../test/abis/latest/SpecDataHolder.json'
+const specDataHolderAbi = specDataHolder.abi
+const specDataHolderBytecode = specDataHolder.bytecode
+
+import badges from '../test/abis/latest/Badges.json'
+const badgesAbi = badges.abi
+const badgesBytecode = badges.bytecode
+
+export default async function testUpgrade(params: any, hre: HardhatRuntimeEnvironment): Promise<void> {
+  // runs npx hardhat compile
+  // without this you'll the code below will be run with stale code
```

Comment:
> // without this you'll the code below will be run with stale code

Can you update the comment to be more legible, please? I think it was a typo :)

Reply:
fixed ... comment not necessary anyway